pkg/operator/ceph/object/user.go

```diff
@@ -100,7 +100,7 @@ func GetUser(c *Context, id string) (*ObjectUser, int, error) {
 
 // CreateUser creates a new user with the information given.
 func CreateUser(c *Context, user ObjectUser) (*ObjectUser, int, error) {
-	logger.Infof("Creating user: %s", user.UserID)
+	logger.Debugf("Creating user: %s", user.UserID)
```

Comment:
Do we still log when the user is created? Or is it all debug now? I think we still need an info message, but maybe it just moved somewhere else.

Reply:
Yes, further up in the main reconcile.